src/metabase/models/recent_views.clj

```diff
@@ -9,7 +9,7 @@
        see: [[update-users-recent-views!]]
```

Comment:
The example above needs to be updated right? Maybe the docstring (or somewhere else in this namespace) could also describe what `context` is used for, particularly what `selection` means since I don't think that's a standard term elsewhere

Reply:
yes! fixed